torch/ao/quantization/__init__.py

```diff
@@ -12,7 +12,6 @@
 from .quantize import *  # noqa: F403
 from .quantize_jit import *  # noqa: F403
 from .stubs import *  # noqa: F403
-from .pt2e.utils import move_model_to_eval
```

Comment:
@andrewor14 i had to make this change otherwise, I was running into circular import due to 
`from torch.ao.quantization.quantizer import QuantizationAnnotation` inside pt2e.utils.py

Reply:
Wait then this is a BC breaking change since we're landing this in 2.1. I think we should find a way to work around this